From 17bca72ce2fba4d00acc5911c0f854b92bca8dce Mon Sep 17 00:00:00 2001
From: Namjae Jeon <linkinjeon@kernel.org>
Date: Wed, 13 Jul 2022 09:24:11 +0900
Subject: [PATCH 02/12] MAINTAINERS: ksmbd: add entry for documentation

Include Documentation/filesystems/cifs/ksmbd.rst in the ksmbd maintainers
file entry.

Signed-off-by: Namjae Jeon <linkinjeon@kernel.org>
Signed-off-by: Steve French <stfrench@microsoft.com>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 08620b9a4..57370f5ec 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -10880,6 +10880,7 @@ R:	Sergey Senozhatsky <senozhatsky@chromium.org>
 L:	linux-cifs@vger.kernel.org
 S:	Maintained
 T:	git git://git.samba.org/ksmbd.git
+F:	Documentation/filesystems/cifs/ksmbd.rst
 F:	fs/ksmbd/
 F:	fs/smbfs_common/
 
-- 
2.37.1.377.g679aad9e82

